platform/default/include/mbgl/storage/offline_database.hpp

```diff
@@ -63,6 +70,7 @@ class OfflineDatabase : private util::noncopyable {
     updateMetadata(const int64_t regionID, const OfflineRegionMetadata&);
 
     std::exception_ptr deleteRegion(OfflineRegion&&);
+    std::exception_ptr invalidateRegion(const int64_t regionID);
```

Comment:
optional nit: no need for `regionID` be const

Reply:
I'm keeping it consistent with other function signatures on the same file.

Reply:
Actually not, I will fix it. :-)